aws/rust-runtime/aws-config/src/ecs.rs

```diff
@@ -451,6 +451,7 @@ mod test {
     use aws_types::os_shim_internal::Env;
     use aws_types::Credentials;
 
+    use aws_smithy_async::rt::sleep::TokioSleep;
```

Comment:
this sleep impl is feature gated right? Does this test work if it's not being compiled with the `rt-tokio` feature?

Reply:
dev dependencies pull in rt-tokio